src/components/advanced-search/advanced-search-form.vue

```diff
@@ -126,6 +126,11 @@ export default class AdvancedSearchForm extends Vue {
     return !!this.startDate && !!this.endDate
   }
 
+  /** True when the form is validating Names. */
+  private get hasAppliedNameRules (): boolean {
+    return !!this.applicantFirstName && !this.applicantLastName
```

Comment:
I'm confused how we got from "the form is validating named" to "if first name and not last name", and also why this means we have applied name rules...?

Reply:
The latter is context when rules are applied. 
Another option is to check the rules array for length but i thought this was a little more explicit

Reply:
The why? I'm not sure, because Designs say so 🤷 hehe

Reply:
> The latter is context when rules are applied.
> Another option is to check the rules array for length but i thought this was a little more explicit

More direct, yes, but also more confusing (and won't work if other rules are added without updating this). In this case I'd prefer the length check -- it would answer all my questions!!!

Reply:
Okie doke, no prob!

Reply:
Updated! 
Sorry it took so long, got pulled into something else at the end of the day there.